sql: schema change acceptance test is timing out

The schema change acceptance test is timing out on azure even with a 6 hour timeout. Probably the same issue some users have reported.

```
[03:02:11][TestRebalance_3To5Small_WithSchemaChanges] + ssh -o ServerAliveInterval=5 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -q -i /root/.ssh/google_compute_engine ubuntu@rebal-3to5s-agitatedalmeida-cockroach-1.eastus.cloudapp.azure.com supervisorctl -c supervisor.conf start block_writer
[03:02:13][TestRebalance_3To5Small_WithSchemaChanges] block_writer: started
[03:02:13][TestRebalance_3To5Small_WithSchemaChanges] + ssh -o ServerAliveInterval=5 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -q -i /root/.ssh/google_compute_engine ubuntu@rebal-3to5s-agitatedalmeida-cockroach-2.eastus.cloudapp.azure.com supervisorctl -c supervisor.conf start block_writer
[03:02:16][TestRebalance_3To5Small_WithSchemaChanges] block_writer: started
[03:02:16][TestRebalance_3To5Small_WithSchemaChanges] + ssh -o ServerAliveInterval=5 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -q -i /root/.ssh/google_compute_engine ubuntu@rebal-3to5s-agitatedalmeida-cockroach-3.eastus.cloudapp.azure.com supervisorctl -c supervisor.conf start block_writer
[03:02:19][TestRebalance_3To5Small_WithSchemaChanges] block_writer: started
[03:02:19][TestRebalance_3To5Small_WithSchemaChanges] + ssh -o ServerAliveInterval=5 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -q -i /root/.ssh/google_compute_engine ubuntu@rebal-3to5s-agitatedalmeida-cockroach-1.eastus.cloudapp.azure.com supervisorctl -c supervisor.conf start photos
[03:02:22][TestRebalance_3To5Small_WithSchemaChanges] photos: started
[03:02:22][TestRebalance_3To5Small_WithSchemaChanges] + ssh -o ServerAliveInterval=5 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -q -i /root/.ssh/google_compute_engine ubuntu@rebal-3to5s-agitatedalmeida-cockroach-2.eastus.cloudapp.azure.com supervisorctl -c supervisor.conf start photos
[03:02:24][TestRebalance_3To5Small_WithSchemaChanges] photos: started
[03:02:24][TestRebalance_3To5Small_WithSchemaChanges] + ssh -o ServerAliveInterval=5 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -q -i /root/.ssh/google_compute_engine ubuntu@rebal-3to5s-agitatedalmeida-cockroach-3.eastus.cloudapp.azure.com supervisorctl -c supervisor.conf start photos
[03:02:27][TestRebalance_3To5Small_WithSchemaChanges] photos: started
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] panic: test timed out after 6h0m0s
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] goroutine 154 [running]:
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] panic(0x16ee820, 0xc420470190)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/runtime/panic.go:500 +0x1a1
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] testing.startAlarm.func1()
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/testing/testing.go:918 +0x10b
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] created by time.goFunc
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/time/sleep.go:154 +0x44
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] goroutine 1 [chan receive, 360 minutes]:
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] testing.(*T).Run(0xc420194240, 0x18ee3c6, 0x29, 0x1a7bc58, 0xc4204cfc01)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/testing/testing.go:647 +0x316
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] testing.RunTests.func1(0xc420194240)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/testing/testing.go:793 +0x6d
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] testing.tRunner(0xc420194240, 0xc420049dd0)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/testing/testing.go:610 +0x81
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] testing.RunTests(0x1a7ec80, 0x23c6300, 0x26, 0x26, 0xc4204cfe28)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/testing/testing.go:799 +0x319
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] testing.(*M).Run(0xc420049ef8, 0x1a7bb58)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/testing/testing.go:743 +0x85
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] github.com/cockroachdb/cockroach/pkg/acceptance.RunTests(0xc420049ef8)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/go/src/github.com/cockroachdb/cockroach/pkg/acceptance/util.go:150 +0x48
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] github.com/cockroachdb/cockroach/pkg/acceptance.TestMain(0xc420049ef8)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/go/src/github.com/cockroachdb/cockroach/pkg/acceptance/main_stub_test.go:42 +0x3d
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] main.main()
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	github.com/cockroachdb/cockroach/pkg/acceptance/_test/_testmain.go:126 +0xc6
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] goroutine 17 [syscall, 360 minutes, locked to thread]:
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] runtime.goexit()
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/runtime/asm_amd64.s:2086 +0x1
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] goroutine 6 [select, 360 minutes, locked to thread]:
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] runtime.gopark(0x1a7f538, 0x0, 0x18c4964, 0x6, 0x18, 0x2)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/runtime/proc.go:259 +0x13a
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] runtime.selectgoImpl(0xc420297f20, 0x0, 0x18)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/runtime/select.go:423 +0x1235
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] runtime.selectgo(0xc420297f20)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/runtime/select.go:238 +0x1c
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] runtime.ensureSigM.func1()
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/runtime/signal1_unix.go:304 +0x2f3
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] runtime.goexit()
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/runtime/asm_amd64.s:2086 +0x1
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] goroutine 19 [chan receive]:
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).flushDaemon(0x28b7520)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1012 +0x77
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] created by github.com/cockroachdb/cockroach/pkg/util/log.init.1
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:583 +0xc4
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] goroutine 20 [chan receive, 360 minutes]:
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0x28b7520)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1039 +0x5b
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] created by github.com/cockroachdb/cockroach/pkg/util/log.init.1
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:584 +0xe8
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] goroutine 21 [syscall, 360 minutes]:
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] os/signal.signal_recv(0x0)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/runtime/sigqueue.go:116 +0x157
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] os/signal.loop()
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/os/signal/signal_unix.go:22 +0x22
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] created by os/signal.init.1
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/os/signal/signal_unix.go:28 +0x41
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] goroutine 23 [chan receive, 360 minutes]:
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] github.com/cockroachdb/cockroach/pkg/acceptance.RunTests.func1()
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/go/src/github.com/cockroachdb/cockroach/pkg/acceptance/util.go:141 +0xb6
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] created by github.com/cockroachdb/cockroach/pkg/acceptance.RunTests
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/go/src/github.com/cockroachdb/cockroach/pkg/acceptance/util.go:149 +0x3a
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] goroutine 24 [semacquire, 342 minutes]:
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] sync.runtime_Semacquire(0xc420470c1c)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/runtime/sema.go:47 +0x30
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] sync.(*WaitGroup).Wait(0xc420470c10)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/sync/waitgroup.go:131 +0x97
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] github.com/cockroachdb/cockroach/pkg/acceptance.(*allocatorTest).runSchemaChanges(0xc42014cd70, 0x2b36ecb3ba18, 0xc4200145b0, 0xc420194300, 0x0, 0x0)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/go/src/github.com/cockroachdb/cockroach/pkg/acceptance/allocator_test.go:247 +0x212
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] github.com/cockroachdb/cockroach/pkg/acceptance.(*allocatorTest).Run(0xc42014cd70, 0x2b36ecb3ba18, 0xc4200145b0, 0xc420194300)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/go/src/github.com/cockroachdb/cockroach/pkg/acceptance/allocator_test.go:195 +0x1068
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] github.com/cockroachdb/cockroach/pkg/acceptance.(*allocatorTest).RunAndCleanup(0xc42014cd70, 0x2b36ecb3ba18, 0xc4200145b0, 0xc420194300)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/go/src/github.com/cockroachdb/cockroach/pkg/acceptance/allocator_test.go:213 +0x13b
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] github.com/cockroachdb/cockroach/pkg/acceptance.TestRebalance_3To5Small_WithSchemaChanges(0xc420194300)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/go/src/github.com/cockroachdb/cockroach/pkg/acceptance/allocator_test.go:501 +0x10a
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] testing.tRunner(0xc420194300, 0x1a7bc58)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/testing/testing.go:610 +0x81
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] created by testing.(*T).Run
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/testing/testing.go:646 +0x2ec
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] goroutine 25 [chan receive, 360 minutes]:
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] github.com/cockroachdb/cockroach/pkg/rpc.NewContext.func1()
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:136 +0x76
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4201ca000, 0xc420150340)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] goroutine 93 [select, 343 minutes]:
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net/http.(*persistConn).writeLoop(0xc420144700)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/http/transport.go:1646 +0x3bd
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] created by net/http.(*Transport).dialConn
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/http/transport.go:1063 +0x50e
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] goroutine 150 [IO wait, 343 minutes]:
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net.runtime_pollWait(0x2b36ecab7640, 0x72, 0x9)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/runtime/netpoll.go:160 +0x59
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net.(*pollDesc).wait(0xc4203f9170, 0x72, 0xc4204cb9d0, 0xc4200140c8)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x38
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net.(*pollDesc).waitRead(0xc4203f9170, 0x23d7ae0, 0xc4200140c8)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x34
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net.(*netFD).Read(0xc4203f9110, 0xc42033f000, 0x1000, 0x1000, 0x0, 0x23d7ae0, 0xc4200140c8)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/fd_unix.go:243 +0x1a1
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net.(*conn).Read(0xc42015c208, 0xc42033f000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/net.go:173 +0x70
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net/http.(*persistConn).Read(0xc42016d600, 0xc42033f000, 0x1000, 0x1000, 0x9ae400, 0xc4204cbb58, 0x62e0fd)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/http/transport.go:1261 +0x154
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] bufio.(*Reader).fill(0xc420178d20)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/bufio/bufio.go:97 +0x10c
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] bufio.(*Reader).Peek(0xc420178d20, 0x1, 0xc4204cbbbd, 0x1, 0x1, 0xc420178d80, 0x0)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/bufio/bufio.go:129 +0x62
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net/http.(*persistConn).readLoop(0xc42016d600)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/http/transport.go:1418 +0x1a1
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] created by net/http.(*Transport).dialConn
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/http/transport.go:1062 +0x4e9
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] goroutine 151 [select, 343 minutes]:
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net/http.(*persistConn).writeLoop(0xc42016d600)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/http/transport.go:1646 +0x3bd
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] created by net/http.(*Transport).dialConn
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/http/transport.go:1063 +0x50e
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] goroutine 202 [chan receive, 342 minutes]:
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] database/sql.(*DB).connectionOpener(0xc4200204d0)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/database/sql/sql.go:730 +0x4a
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] created by database/sql.Open
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/database/sql/sql.go:493 +0x1e9
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] goroutine 92 [IO wait, 343 minutes]:
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net.runtime_pollWait(0x2b36ecab77c0, 0x72, 0x5)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/runtime/netpoll.go:160 +0x59
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net.(*pollDesc).wait(0xc4203ce4c0, 0x72, 0xc4202929d0, 0xc4200140c8)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x38
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net.(*pollDesc).waitRead(0xc4203ce4c0, 0x23d7ae0, 0xc4200140c8)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x34
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net.(*netFD).Read(0xc4203ce460, 0xc42006e000, 0x1000, 0x1000, 0x0, 0x23d7ae0, 0xc4200140c8)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/fd_unix.go:243 +0x1a1
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net.(*conn).Read(0xc42014a1a0, 0xc42006e000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/net.go:173 +0x70
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net/http.(*persistConn).Read(0xc420144700, 0xc42006e000, 0x1000, 0x1000, 0x9ae400, 0xc420292b58, 0x62e0fd)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/http/transport.go:1261 +0x154
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] bufio.(*Reader).fill(0xc42043d9e0)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/bufio/bufio.go:97 +0x10c
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] bufio.(*Reader).Peek(0xc42043d9e0, 0x1, 0xc420292bbd, 0x1, 0x1, 0xc42043da40, 0x0)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/bufio/bufio.go:129 +0x62
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net/http.(*persistConn).readLoop(0xc420144700)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/http/transport.go:1418 +0x1a1
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] created by net/http.(*Transport).dialConn
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/http/transport.go:1062 +0x4e9
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] goroutine 94 [IO wait, 343 minutes]:
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net.runtime_pollWait(0x2b36ecab7880, 0x72, 0x7)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/runtime/netpoll.go:160 +0x59
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net.(*pollDesc).wait(0xc420446450, 0x72, 0xc4202919d0, 0xc4200140c8)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x38
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net.(*pollDesc).waitRead(0xc420446450, 0x23d7ae0, 0xc4200140c8)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x34
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net.(*netFD).Read(0xc4204463f0, 0xc4203aa000, 0x1000, 0x1000, 0x0, 0x23d7ae0, 0xc4200140c8)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/fd_unix.go:243 +0x1a1
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net.(*conn).Read(0xc42014a1b0, 0xc4203aa000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/net.go:173 +0x70
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net/http.(*persistConn).Read(0xc420144900, 0xc4203aa000, 0x1000, 0x1000, 0x9ae400, 0xc420291b58, 0x62e0fd)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/http/transport.go:1261 +0x154
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] bufio.(*Reader).fill(0xc42043dc80)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/bufio/bufio.go:97 +0x10c
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] bufio.(*Reader).Peek(0xc42043dc80, 0x1, 0xc420291bbd, 0x1, 0x1, 0xc42043dce0, 0x0)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/bufio/bufio.go:129 +0x62
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net/http.(*persistConn).readLoop(0xc420144900)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/http/transport.go:1418 +0x1a1
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] created by net/http.(*Transport).dialConn
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/http/transport.go:1062 +0x4e9
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] goroutine 95 [select, 343 minutes]:
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net/http.(*persistConn).writeLoop(0xc420144900)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/http/transport.go:1646 +0x3bd
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] created by net/http.(*Transport).dialConn
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/http/transport.go:1063 +0x50e
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] goroutine 148 [IO wait, 343 minutes]:
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net.runtime_pollWait(0x2b36ecab7700, 0x72, 0x8)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/runtime/netpoll.go:160 +0x59
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] Test ended in panic.
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net.(*pollDesc).wait(0xc4203f8fb0, 0x72, 0xc4204ca9d0, 0xc4200140c8)
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] 	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x38
[08:44:47][TestRebalance_3To5Small_WithSchemaChanges] net.(*pollDesc).waitRead(0xc4203f8fb0, 0x23d7ae0, 0xc4200140c8)
[08:44:47]	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x34
[08:44:47]net.(*netFD).Read(0xc4203f8f50, 0xc420091000, 0x1000, 0x1000, 0x0, 0x23d7ae0, 0xc4200140c8)
[08:44:47]	/usr/local/go/src/net/fd_unix.go:243 +0x1a1
[08:44:47]net.(*conn).Read(0xc42015c200, 0xc420091000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
[08:44:47]	/usr/local/go/src/net/net.go:173 +0x70
[08:44:47]net/http.(*persistConn).Read(0xc420144a00, 0xc420091000, 0x1000, 0x1000, 0x9ae400, 0xc4204cab58, 0x62e0fd)
[08:44:47]	/usr/local/go/src/net/http/transport.go:1261 +0x154
[08:44:47]bufio.(*Reader).fill(0xc420178b40)
[08:44:47]	/usr/local/go/src/bufio/bufio.go:97 +0x10c
[08:44:47]bufio.(*Reader).Peek(0xc420178b40, 0x1, 0xc4204cabbd, 0x1, 0x1, 0xc420178ba0, 0x0)
[08:44:47]	/usr/local/go/src/bufio/bufio.go:129 +0x62
[08:44:47]net/http.(*persistConn).readLoop(0xc420144a00)
[08:44:47]	/usr/local/go/src/net/http/transport.go:1418 +0x1a1
[08:44:47]created by net/http.(*Transport).dialConn
[08:44:47]	/usr/local/go/src/net/http/transport.go:1062 +0x4e9
[08:44:47]
[08:44:47]goroutine 149 [select, 343 minutes]:
[08:44:47]net/http.(*persistConn).writeLoop(0xc420144a00)
[08:44:47]	/usr/local/go/src/net/http/transport.go:1646 +0x3bd
[08:44:47]created by net/http.(*Transport).dialConn
[08:44:47]	/usr/local/go/src/net/http/transport.go:1063 +0x50e
[08:44:47]
[08:44:47]goroutine 181 [IO wait, 343 minutes]:
[08:44:47]net.runtime_pollWait(0x2b36ecab7940, 0x72, 0xa)
[08:44:47]	/usr/local/go/src/runtime/netpoll.go:160 +0x59
[08:44:47]net.(*pollDesc).wait(0xc4204466f0, 0x72, 0xc4202959d0, 0xc4200140c8)
[08:44:47]	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x38
[08:44:47]net.(*pollDesc).waitRead(0xc4204466f0, 0x23d7ae0, 0xc4200140c8)
[08:44:47]	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x34
[08:44:47]net.(*netFD).Read(0xc420446690, 0xc4204e7000, 0x1000, 0x1000, 0x0, 0x23d7ae0, 0xc4200140c8)
[08:44:47]	/usr/local/go/src/net/fd_unix.go:243 +0x1a1
[08:44:47]net.(*conn).Read(0xc42014a3e8, 0xc4204e7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
[08:44:47]	/usr/local/go/src/net/net.go:173 +0x70
[08:44:47]net/http.(*persistConn).Read(0xc420144b00, 0xc4204e7000, 0x1000, 0x1000, 0x9ae400, 0xc420295b58, 0x62e0fd)
[08:44:47]	/usr/local/go/src/net/http/transport.go:1261 +0x154
[08:44:47]bufio.(*Reader).fill(0xc420183020)
[08:44:47]	/usr/local/go/src/bufio/bufio.go:97 +0x10c
[08:44:47]bufio.(*Reader).Peek(0xc420183020, 0x1, 0xc420295bbd, 0x1, 0x1, 0xc420183080, 0x0)
[08:44:47]	/usr/local/go/src/bufio/bufio.go:129 +0x62
[08:44:47]net/http.(*persistConn).readLoop(0xc420144b00)
[08:44:47]	/usr/local/go/src/net/http/transport.go:1418 +0x1a1
[08:44:47]created by net/http.(*Transport).dialConn
[08:44:47]	/usr/local/go/src/net/http/transport.go:1062 +0x4e9
[08:44:47]
[08:44:47]goroutine 182 [select, 343 minutes]:
[08:44:47]net/http.(*persistConn).writeLoop(0xc420144b00)
[08:44:47]	/usr/local/go/src/net/http/transport.go:1646 +0x3bd
[08:44:47]created by net/http.(*Transport).dialConn
[08:44:47]	/usr/local/go/src/net/http/transport.go:1063 +0x50e
[08:44:47]
[08:44:47]goroutine 203 [IO wait, 342 minutes]:
[08:44:47]net.runtime_pollWait(0x2b36ecab74c0, 0x72, 0xb)
[08:44:47]	/usr/local/go/src/runtime/netpoll.go:160 +0x59
[08:44:47]net.(*pollDesc).wait(0xc4203ce290, 0x72, 0xc4205a1808, 0xc4200140c8)
[08:44:47]	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x38
[08:44:47]net.(*pollDesc).waitRead(0xc4203ce290, 0x23d7ae0, 0xc4200140c8)
[08:44:47]	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x34
[08:44:47]net.(*netFD).Read(0xc4203ce230, 0xc4205a2000, 0x1000, 0x1000, 0x0, 0x23d7ae0, 0xc4200140c8)
[08:44:47]	/usr/local/go/src/net/fd_unix.go:243 +0x1a1
[08:44:47]net.(*conn).Read(0xc420288040, 0xc4205a2000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
[08:44:47]	/usr/local/go/src/net/net.go:173 +0x70
[08:44:47]bufio.(*Reader).fill(0xc42043d920)
[08:44:47]	/usr/local/go/src/bufio/bufio.go:97 +0x10c
[08:44:47]bufio.(*Reader).Read(0xc42043d920, 0xc4203bc020, 0x5, 0x200, 0x2, 0xc4205a19b0, 0x0)
[08:44:47]	/usr/local/go/src/bufio/bufio.go:209 +0x1bc
[08:44:47]io.ReadAtLeast(0x23cf660, 0xc42043d920, 0xc4203bc020, 0x5, 0x200, 0x5, 0xc4205488a0, 0x20, 0x18)
[08:44:47]	/usr/local/go/src/io/io.go:307 +0xa4
[08:44:47]io.ReadFull(0x23cf660, 0xc42043d920, 0xc4203bc020, 0x5, 0x200, 0xc420138c00, 0x55, 0x2b36eca4f000)
[08:44:47]	/usr/local/go/src/io/io.go:325 +0x58
[08:44:47]github.com/cockroachdb/cockroach/vendor/github.com/lib/pq.(*conn).recvMessage(0xc4203bc000, 0xc4205488a0, 0x16f51e0, 0x7ccb01, 0x28f3458)
[08:44:47]	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/lib/pq/conn.go:926 +0x117
[08:44:47]github.com/cockroachdb/cockroach/vendor/github.com/lib/pq.(*conn).recv1Buf(0xc4203bc000, 0xc4205488a0, 0x0)
[08:44:47]	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/lib/pq/conn.go:976 +0x39
[08:44:47]github.com/cockroachdb/cockroach/vendor/github.com/lib/pq.(*conn).recv1(0xc4203bc000, 0xc4205a1bb0, 0x50)
[08:44:47]	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/lib/pq/conn.go:997 +0x87
[08:44:47]github.com/cockroachdb/cockroach/vendor/github.com/lib/pq.(*conn).simpleExec(0xc4203bc000, 0xc42001e230, 0x4f, 0xc4205a1c80, 0xc4205a1c30, 0x637e89, 0x1763c80, 0x18ac780, 0xc4205a1c01)
[08:44:47]	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/lib/pq/conn.go:599 +0x1ef
[08:44:47]github.com/cockroachdb/cockroach/vendor/github.com/lib/pq.(*conn).Exec(0xc4203bc000, 0xc42001e230, 0x4f, 0x28f3458, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
[08:44:47]	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/lib/pq/conn.go:846 +0x2e2
[08:44:47]database/sql.(*DB).exec(0xc4200204d0, 0xc42001e230, 0x4f, 0x0, 0x0, 0x0, 0x78f101, 0x0, 0x0, 0x0, ...)
[08:44:47]	/usr/local/go/src/database/sql/sql.go:1036 +0x4ca
[08:44:47]database/sql.(*DB).Exec(0xc4200204d0, 0xc42001e230, 0x4f, 0x0, 0x0, 0x0, 0x1, 0x8, 0x28b8480, 0xc42036fba0)
[08:44:47]	/usr/local/go/src/database/sql/sql.go:1010 +0xa2
[08:44:47]github.com/cockroachdb/cockroach/pkg/acceptance.(*allocatorTest).runSchemaChanges.func2(0xc4202acea0, 0x2, 0x2, 0x2b36ecb3ba18, 0xc4200145b0, 0xc4200204d0, 0xc420194300, 0xc420470c10, 0x0)
[08:44:47]	/go/src/github.com/cockroachdb/cockroach/pkg/acceptance/allocator_test.go:237 +0x234
[08:44:47]created by github.com/cockroachdb/cockroach/pkg/acceptance.(*allocatorTest).runSchemaChanges
[08:44:47]	/go/src/github.com/cockroachdb/cockroach/pkg/acceptance/allocator_test.go:245 +0x1ec
[08:44:47]
[08:44:47]goroutine 204 [IO wait, 342 minutes]:
[08:44:47]net.runtime_pollWait(0x2b36ecab7580, 0x72, 0xc)
[08:44:47]	/usr/local/go/src/runtime/netpoll.go:160 +0x59
[08:44:47]net.(*pollDesc).wait(0xc4203ce3e0, 0x72, 0xc420355808, 0xc4200140c8)
[08:44:47]	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x38
[08:44:47]net.(*pollDesc).waitRead(0xc4203ce3e0, 0x23d7ae0, 0xc4200140c8)
[08:44:47]	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x34
[08:44:47]net.(*netFD).Read(0xc4203ce380, 0xc4205a3000, 0x1000, 0x1000, 0x0, 0x23d7ae0, 0xc4200140c8)
[08:44:47]	/usr/local/go/src/net/fd_unix.go:243 +0x1a1
[08:44:47]net.(*conn).Read(0xc420288048, 0xc4205a3000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
[08:44:47]	/usr/local/go/src/net/net.go:173 +0x70
[08:44:47]bufio.(*Reader).fill(0xc42043d980)
[08:44:47]	/usr/local/go/src/bufio/bufio.go:97 +0x10c
[08:44:47]bufio.(*Reader).Read(0xc42043d980, 0xc42025e7a0, 0x5, 0x200, 0x2, 0xc4203559b0, 0x0)
[08:44:47]	/usr/local/go/src/bufio/bufio.go:209 +0x1bc
[08:44:47]io.ReadAtLeast(0x23cf660, 0xc42043d980, 0xc42025e7a0, 0x5, 0x200, 0x5, 0xc420548880, 0x20, 0x18)
[08:44:47]	/usr/local/go/src/io/io.go:307 +0xa4
[08:44:47]io.ReadFull(0x23cf660, 0xc42043d980, 0xc42025e7a0, 0x5, 0x200, 0xc420138c00, 0x36, 0x2b36eca4f000)
[08:44:47]	/usr/local/go/src/io/io.go:325 +0x58
[08:44:47]github.com/cockroachdb/cockroach/vendor/github.com/lib/pq.(*conn).recvMessage(0xc42025e780, 0xc420548880, 0x16f51e0, 0x7ccb01, 0x28f3458)
[08:44:47]	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/lib/pq/conn.go:926 +0x117
[08:44:47]github.com/cockroachdb/cockroach/vendor/github.com/lib/pq.(*conn).recv1Buf(0xc42025e780, 0xc420548880, 0x0)
[08:44:47]	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/lib/pq/conn.go:976 +0x39
[08:44:47]github.com/cockroachdb/cockroach/vendor/github.com/lib/pq.(*conn).recv1(0xc42025e780, 0xc420355bb0, 0x31)
[08:44:47]	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/lib/pq/conn.go:997 +0x87
[08:44:47]github.com/cockroachdb/cockroach/vendor/github.com/lib/pq.(*conn).simpleExec(0xc42025e780, 0xc420218e10, 0x30, 0xc420355c80, 0xc420355c30, 0x637e89, 0x1763c80, 0x18ac780, 0xc420355c01)
[08:44:47]	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/lib/pq/conn.go:599 +0x1ef
[08:44:47]github.com/cockroachdb/cockroach/vendor/github.com/lib/pq.(*conn).Exec(0xc42025e780, 0xc420218e10, 0x30, 0x28f3458, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
[08:44:47]	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/lib/pq/conn.go:846 +0x2e2
[08:44:47]database/sql.(*DB).exec(0xc4200204d0, 0xc420218e10, 0x30, 0x0, 0x0, 0x0, 0x78f101, 0x0, 0x0, 0x0, ...)
[08:44:47]	/usr/local/go/src/database/sql/sql.go:1036 +0x4ca
[08:44:47]database/sql.(*DB).Exec(0xc4200204d0, 0xc420218e10, 0x30, 0x0, 0x0, 0x0, 0x1, 0xc42026ff98, 0x9a81cc, 0xc420179200)
[08:44:47]	/usr/local/go/src/database/sql/sql.go:1010 +0xa2
[08:44:47]github.com/cockroachdb/cockroach/pkg/acceptance.(*allocatorTest).runSchemaChanges.func2(0xc4202acea0, 0x2, 0x2, 0x2b36ecb3ba18, 0xc4200145b0, 0xc4200204d0, 0xc420194300, 0xc420470c10, 0x1)
[08:44:47]	/go/src/github.com/cockroachdb/cockroach/pkg/acceptance/allocator_test.go:237 +0x234
[08:44:47]created by github.com/cockroachdb/cockroach/pkg/acceptance.(*allocatorTest).runSchemaChanges
[08:44:47]	/go/src/github.com/cockroachdb/cockroach/pkg/acceptance/allocator_test.go:245 +0x1ec
[08:44:47]make: *** [test] Error 1
[08:44:48]Process exited with code 2
```

--- a/pkg/acceptance/allocator_test.go
+++ b/pkg/acceptance/allocator_test.go
@@ -72,8 +72,6 @@ import (
 	"github.com/montanaflynn/stats"
 	"github.com/pkg/errors"
 
-	"sync"
-
 	"github.com/cockroachdb/cockroach/pkg/acceptance/terrafarm"
 	"github.com/cockroachdb/cockroach/pkg/base"
 	"github.com/cockroachdb/cockroach/pkg/server/serverpb"
@@ -227,24 +225,28 @@ func (at *allocatorTest) runSchemaChanges(ctx context.Context, t *testing.T) err
 		"ALTER TABLE %s ADD COLUMN newcol DECIMAL DEFAULT (DECIMAL '1.4')",
 		"CREATE INDEX foo ON %s (block_id)",
 	}
-	var wg sync.WaitGroup
-	wg.Add(len(schemaChanges))
+
+	errChan := make(chan error)
 	for i := range schemaChanges {
 		go func(i int) {
 			start := timeutil.Now()
 			cmd := fmt.Sprintf(schemaChanges[i], tableName)
 			log.Infof(ctx, "starting schema change: %s", cmd)
 			if _, err := db.Exec(cmd); err != nil {
-				log.Infof(ctx, "hit schema change error: %s, for %s, in %s", err, cmd, timeutil.Since(start))
-				t.Error(err)
+				errChan <- errors.Errorf("hit schema change error: %s, for %s, in %s", err, cmd, timeutil.Since(start))
 				return
 			}
 			log.Infof(ctx, "completed schema change: %s, in %s", cmd, timeutil.Since(start))
-			wg.Done()
+			errChan <- nil
 			// TODO(vivek): Monitor progress of schema changes and log progress.
 		}(i)
 	}
-	wg.Wait()
+
+	for range schemaChanges {
+		if err := <-errChan; err != nil {
+			t.Fatal(err)
+		}
+	}
 
 	log.Info(ctx, "validate applied schema changes")
 	if err := at.ValidateSchemaChanges(ctx, t); err != nil {

